src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -86,7 +83,7 @@
  */
 public class Database {
 
-    static final Log log = LogFactory.getLog(Database.class);
+    static final Logger log = Logger.getLogger(Database.class.getName());
```

Comment:
I think you might want to use `Database.class.getCanonicalName()` here instead of `Database.class.getName()` since it seems to be more appropriate for logging operations.  See:  http://stackoverflow.com/questions/15202997/what-is-the-difference-between-canonical-name-simple-name-and-class-name-in-jav.